src/test/suite/completion.test.ts

```diff
@@ -107,6 +109,23 @@ describe("Should do completion", () => {
 
     await makeAndAssertFollowingChange();
   });
+  it.only("should skip completion request on midline invalid position", async () => {
+    await openADocWith(" s");
+
+    await moveToStartOfLinePosition();
+
+    mockInlineResponse();
+
+    await makeAChange("console");
+
+    await triggerInline();
+
+    await emulationUserInteraction();
+
+    await acceptInline();
```

Comment:
@dimacodota is there a nicer way to get the current presented completions in the document?

Reply:
there is no API to get the rendered  inline suggestion, so only accept and assert the inserted text

Reply:
I thought so :\
ready for review